worker/games.py

```diff
@@ -93,13 +93,31 @@ def verify_signature(engine, signature, remote, payload, concurrency):
 
   return bench_nps
 
+def cleanup(item, testing_dir):
+  """Remove corrupted opening books"""
+  target_path = os.path.join(testing_dir, item)
+  if os.path.exists(target_path):
+    stat_info = os.stat(target_path)
+  else:
+    return True
+  if stat_info.st_size == 0:
+    os.remove(target_path)
+    return False
```

Comment:
Return false when the file does no longer exist. I do not understand the logic related to the previous comment. I would expect the same return value, because in both cases the file must be down loaded?

Reply:
The logic is the following: if I have a file 0 in the moment in which I call the cleanup function in the `while` clause, I will remove it. Next time when I go back I will have removed it, so I return `True` and can go on and write down the the file.

The problem I realize now: this will not prevent the worker from writing a 0-bytes file, but it will try to fix it next time. I admit that this is suboptimal and somewhat obscure.